lib/glueby/contract/payment.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Glueby
+  module Contract
+    # This class can send TPC between wallets.
+    #
+    # Examples:
+    #
+    # sender = Glueby::Wallet.load("wallet_id")
+    # reciever = Glueby::Wallet.load("wallet_id")
+    #                      or
+    #            Glueby::Wallet.create
+    #
+    # Balance of sender and reciever before send
+    # sender.internal_wallet.balance
+    # => 100_000(tapyrus)
+    # reciever.internal_wallet.balance
+    # => 0(tapyrus)
+    #
+    # Send
+    # Payment.transfer(sender_wallet: sender, receiver_wallet: reciever, amount: 10_000)
+    # sender.internal_wallet.balance
+    # => 90_000
+    # reciever.internal_wallet.balance
+    # => 10_000
+    #
+    class Payment
+      include Glueby::Contract::TxBuilder
```

Comment:
This `include` statement is not needed because this class does not provide any instance methods.

Reply:
Remove this include. Thank you.